docs/supported_platforms.md

```diff
@@ -17,6 +16,10 @@ unsupported platforms.
 Our policy regarding supported platforms can be found in our ["stability
 guarantees"](semver.md).
 
+We require that you use the most recent kernel available for your platform.
```

Comment:
OTOH we can't control what GH actions does; unless we find some documentation about what they promise re kernel versions, maybe something more like "we support the latest default kernel for each supported platform, but in practice only test on the kernel versions that GitHub actions makes available; today this is the latest Ubuntu HWE kernel."

Reply:
> It looks like both the ubuntu 22.04 and 20.04 images are using "5.15.0-1034-azure", which looks like it would indeed be a HWE kernel since it would otherwise be 5.13 for 20.04.

Kernel 5.13 is only for Ubuntu 20.04.4. People may be running 20.04.0 which uses the GA kernel 5.4.

Reply:
I changed it to the following, lmk what you think:

"Some Linux distributions support multiple kernel versions, for example an older General Availability (GA) kernel and newer hardware-enablement (HWE) kernels. We will try to allow Shadow to run on the oldest kernel supported on each distribution (the GA kernel), but we can only commit to officially supporting the newest kernel supported on each distribution (the latest HWE kernel) due to restrictions in our CI testing infrastructure."

Reply:
> but we can only commit to officially supporting the newest kernel supported on each distribution (the latest HWE kernel) due to restrictions in our CI testing infrastructure.

We're only able to test on Ubuntu's HWE kernel in particular; e.g. we have no way of testing on Debian kernels.

Maybe it's better not to say something as strong as "commit", but instead something like "but we are currently only able to regularly test on the latest Ubuntu kernels, since that's what GitHub Actions provides."

Reply:
Changed it to your quote.